src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -683,7 +698,7 @@ def _save_or_select_or_merge(
         sparsity_overlap=0.75,
         verbose=False,
         new_unit_ids=None,
-        **job_kwargs,
+        **kwargs,
```

Comment:
I would create a clear spearate dict here like `storage_option={}`.
we do this to create an analyzer we should do the same for the save_as() no ?